deploy/namespace.yaml

```diff
@@ -2,3 +2,5 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: windows-machine-config-operator
+  labels:
+    openshift.io/run-level: "1"
```

Comment:
> I can give you a label selector to use temporarily that is not something you can GA with that will skip SCC. any namespace with a label "openshift.io/run-level = 1" will skip SCC.  But you still have a general problem with providing a way to safely validate pods that want to run on windows and SCC must not be skipped on user requests if the pod is a linux pod.

This is from our discussion of the Windows webhook. This does not seem to be the correct way to go about this

Reply:
> providing a way to safely validate pods that want to run on windows and SCC must not be skipped on **user** requests if the pod is a linux pod.

My understanding of the above statement is - it is correct for user request pods not the cluster ones. For the Windows specific pods that run in any namespace we shouldn't add this label or ask user to add this label for the namespace.

Reply:
To be clear, only `windows-machine-config-operator` namespace should have this label and the WMCO pod should be the only pod running in this namespace.

Reply:
I am not too familar with the terminology, so I did not focus on the `user requests` part. That makes things better.

I am still hesitant about this solution. Instead of removing just the `hostnetwork` blocking you are removing all SCC. It seems like we should be more focused on what we are removing. I am worried about vulnerabilities that could arise from this.

Reply:
I'll see if I can associate WMCO service account with hostNetwork SCC and see if it works.

Reply:
For my understanding, you are using the label just so that the pod can run on the `host-network`. If so, what do other components that need to run on the `host-network` do?

Reply:
Atleast MCO uses this label on the namespace they need.